src/components/HelpModal/HelpModal.tsx

```diff
@@ -87,7 +87,7 @@ export const HelpModal: FunctionComponent<{
 
   const onPressAskQuestion = (): void => {
     if (webViewRef.current) {
-      const redirectTo = `window.location = "${SUPPORT_URL}"`;
+      const redirectTo = `${SUPPORT_URL}`;
```

Comment:
dont need the window.location anymore if we plan to direct them to their mail app